index.html

```diff
@@ -35,7 +38,55 @@ <h2 class="card__heading">Question:</h2>
       </section>
 
       <section class="card">
-        <button class="card__bookmark">Bookmark</button>
+        <button data-js="button-fill" class="card__bookmark">Bookmark</button>
+        <h2 class="card__heading">Question:</h2>
+        <p class="card__question">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Modi earum
+          voluptas quisquam numquam aut nostrum, voluptatum voluptatibus nobis.
+        </p>
+        <button data-js="answer-button" class="card__answer-button">
+          Show Answer
+        </button>
+        <p data-js="answer-text" class="card__answer dp-hide">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </p>
+        <ul role="list" class="card__ul">
+          <li class="card__ul__li">Tag 1</li>
+          <li class="card__ul__li">Tag 2</li>
+          <li class="card__ul__li">Tag 3</li>
+          <li class="card__ul__li">Tag 4</li>
+        </ul>
+      </section>
+
+      <section class="card">
+        <button data-js="button-fill" class="card__bookmark">Bookmark</button>
+        <h2 class="card__heading">Question:</h2>
+        <p class="card__question">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Modi earum
+          voluptas quisquam numquam aut nostrum, voluptatum voluptatibus nobis.
+        </p>
+        <button data-js="answer-button" class="card__answer-button">
+          Show Answer
+        </button>
+        <p data-js="answer-text" class="card__answer dp-hide">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </p>
+        <ul role="list" class="card__ul">
+          <li class="card__ul__li">Tag 1</li>
+          <li class="card__ul__li">Tag 2</li>
+          <li class="card__ul__li">Tag 3</li>
+          <li class="card__ul__li">Tag 4</li>
+        </ul>
+      </section>
+    </main>
+
+    <main data-js="bookmarks" class="app-grid__scrollarea dp-hide">
```

Comment:
I think you should have only one main with four different divs (at least i did used divs 😊).
But in general main should be only once in an html as far i understood. 😊

Reply:
Thank you for your feedback :) I will check this later :)